server/business-process-query/get-status-history-date-map.js

```diff
@@ -43,7 +43,9 @@ Date map layout:
 		// 2.5 On business process - At withdrawn or closed
 		withdrawn: sum,
 		// 2.6 On business process - At rejected
-		rejected: sum
+		rejected: sum,
+		// Notice: Not on business process, but on actual certificate approved
+		approved: sum
```

Comment:
It would mean a great deal to me if you could keep using that numbering I introduced for easier debugging.